konig-data-catalog/src/main/java/io/konig/datacatalog/PropertyPage.java

```diff
@@ -91,8 +92,9 @@ private void setEquivalentPath(PropertyRequest request) throws DataCatalogExcept
 			
 			List<PathElementView> list = new ArrayList<>();
 			for (Step step : path.asList()) {
-				
-				list.add(pathElement(request, step));
+				if(!(step instanceof InStep)){

```

Comment:
We should not skip InStep elements as shown here.  Instead, the `pathElement` method should handle the InStep element gracefully (and not throw an exception).  An InStep element should be handled in the same way as an OutStep element but with the caret ("^") as the operator.